app/webpack.config.js

```diff
@@ -0,0 +1,71 @@
+const path = require("path");
+const webpack = require("webpack");
+const CaseSensitivePathsPlugin = require("case-sensitive-paths-webpack-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = {
+  context: path.resolve(__dirname, "src"),
+  entry: "./index.js",
+  output: {
+    filename: "bundle.js",
+    path: path.resolve(__dirname, "dist")
+  },
+  devServer: {
+    contentBase: path.resolve(__dirname, "dist")
+  },
+  resolve: {
+    extensions: [".tsx", ".jsx", ".ts", ".js"]
+  },
+  optimization: {
+    splitChunks: {
+      cacheGroups: {
+        commons: {
+          name: "commons",
+          chunks: "all",
+          minSize: 0,
+          minChunks: 2
+        }
+      }
+    }
+  },
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        loader: "ts-loader",
+        exclude: /node-modules/,
+        options: {
+          useCache: true
+        }
+      },
+      {
+        test: /\*.(jsx|js)/,
```

Comment:
You test loader patterns are:
1. /\*.(jsx|js)/
2. /\.tsx?$/
3. /\.s[ac]ss$/i
Please, make them look similar :)